src/fabric/api/fixed-asset-base/go/contracts/FixedAssetContract.go

```diff
@@ -125,6 +125,38 @@ func (contract *FixedAssetContract) GetAssetsFromBatch(ctx utils.Context, batch
 	return &retArr, nil
 }
 
+//Delete an Asset from the registry that was created by createAsset
+func (s *FixedAssetContract) DeleteAsset(ctx utils.Context, uuid string) error {
+	fmt.Printf("Entering deleteAsset")
```

Comment:
Should use Println rather than Printf otherwise it will put everything on the same line